tutorials/text/annotations.py

```diff
@@ -429,89 +415,87 @@
+   above example) is determined before *an2* gets drawn. Usually, this means
+   that *an2* needs to be drawn after *an1*.
 
+3. A callable object that takes the renderer instance as single argument, and
+   returns an instance of either `.Transform` or `.BboxBase`.  The return value
+   is then handled as in (1), for transforms, or in (2), for bboxes.  For
+   example, ::
 
- 3. A callable object that returns an instance of either
-    :class:`~matplotlib.transforms.BboxBase` or
-    :class:`~matplotlib.transforms.Transform`. If a transform is
-    returned, it is the same as 1 and if a bbox is returned, it is the same
-    as 2.  The callable object should take a single argument of the
-    renderer instance. For example, the following two commands give
-    identical results ::
+     an2 = ax.annotate("Test 2", xy=(1, 0.5), xycoords=an1,
+                       xytext=(30, 0), textcoords="offset points")
 
-      an2 = ax.annotate("Test 2", xy=(1, 0.5), xycoords=an1,
-                        xytext=(30, 0), textcoords="offset points")
-      an2 = ax.annotate("Test 2", xy=(1, 0.5), xycoords=an1.get_window_extent,
-                        xytext=(30, 0), textcoords="offset points")
+   is identical to::
 
+     an2 = ax.annotate("Test 2", xy=(1, 0.5), xycoords=an1.get_window_extent,
+                       xytext=(30, 0), textcoords="offset points")
 
- 4. A tuple of two coordinate specifications. The first item is for the
-    x-coordinate and the second is for the y-coordinate. For example, ::
+4. A pair of coordinate specifications -- the first for the x-coordinate, and
+   the second is for the y-coordinate; e.g. ::
 
-      annotate("Test", xy=(0.5, 1), xycoords=("data", "axes fraction"))
+     annotate("Test", xy=(0.5, 1), xycoords=("data", "axes fraction"))
 
-    0.5 is in data coordinates, and 1 is in normalized axes coordinates.
-    You may use an artist or transform as with a tuple. For example,
+   Here, 0.5 is in data coordinates, and 1 is in normalized axes coordinates.
+   Each of the coordinate specifications can also be an artist or a transform.
+   For example,
 
-    .. figure:: ../../gallery/userdemo/images/sphx_glr_annotate_simple_coord02_001.png
-       :target: ../../gallery/userdemo/annotate_simple_coord02.html
-       :align: center
-       :scale: 50
+   .. figure:: ../../gallery/userdemo/images/sphx_glr_annotate_simple_coord02_001.png
+      :target: ../../gallery/userdemo/annotate_simple_coord02.html
+      :align: center
+      :scale: 50
 
-       Annotation with Simple Coordinates 2
+      Annotation with Simple Coordinates 2
 
- 5. Sometimes, you want your annotation with some "offset points", not from the
-    annotated point but from some other point.
-    :class:`~matplotlib.text.OffsetFrom` is a helper class for such cases.
+5. Sometimes, you want your annotation with some "offset points", not from the
+   annotated point but from some other point.  `.text.OffsetFrom` is a helper
```

Comment:
Actually this specific class is obscure enough that I intentionally decided to mention the parent module, even though this may appear slightly inconsistent.  I can remove it if you feel strongly about it, though.

Reply:
Fair enough